learning-team/2020-09/2020-09-24.md

```diff
@@ -0,0 +1,109 @@
+# September 24, 2020
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [ ] Amy Lam (AL)
+- [x] Chris Manson (CM)
+- [x] Isaac Lee (IL)
+- [x] Jared Galanis (JG)
+- [x] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [ ] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [x] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [x] fozy81
+
+## Weekly review
+
+### Review action items (weekly)
+See: [Action items board](https://github.com/orgs/ember-learn/projects/47)
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/33)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+### What makes a quorum? (general discussion, 5 min)
+
+There might not have been a prior definition for a quorum. We will want to be specific.
+
+Let's decide on the definition between tomorrow and next meeting?
+
+### Follow-up to Ember Guides feedback (IL, 5-10 min)
+
+If there is a quorum, let's follow up on last week's discussion on [conveying better a change in code](https://github.com/emberjs/core-notes/blob/master/learning-team/2020-09/2020-09-17.md#feedback-for-ember-guides-il-10-min)? Additional feedback can be found in https://github.com/ember-learn/guides-source/issues/1534.
+
+- Move to next week
```

Comment:
The Learning Team decided to move the majority of discussion items to next week's meeting so that we have more time to prepare for Hacktoberfest.

When you create a new Markdown file and issue for October 1, please carry over these items.